observatory-platform/observatory/platform/telescopes/telescope.py

```diff
@@ -88,6 +89,16 @@ def add_setup_task_chain(self, funcs: List[Callable]):
         """
         pass
 
+    @abstractmethod
+    def add_sensors(self, sensors: List[BaseSensorOperator]):
```

Comment:
It would be good to also add an interface `add_sensor` so that you can add a single sensor at a time if you wish.